git_machete/cli.py

```diff
@@ -354,192 +566,154 @@ def check_required_param(in_args: List[str], allowed_values_string: str) -> str:
                 machete_client.unset_fork_point_override(branch)
             else:
                 print(machete_client.fork_point(branch, use_overrides=True))
-        elif cmd in ("g", "go"):
-            param = check_required_param(parse_options(args),
-                                         allowed_directions(allow_current=False))
+        elif cmd == "go":
             machete_client.read_definition_file()
             git.expect_no_operation_in_progress()
             current_branch = git.get_current_branch()
-            dest = machete_client.parse_direction(param, current_branch, allow_current=False,
-                                                  down_pick_mode=True)
+            dest = machete_client.parse_direction(
+                parsed_cli.direction, current_branch, allow_current=False, down_pick_mode=True)
             if dest != current_branch:
                 git.checkout(dest)
         elif cmd == "github":
-            github_allowed_subcommands = "anno-prs|checkout-prs|create-pr|retarget-pr"
-            list_args = parse_options(args) if args[0] == 'checkout-prs' \
-                else check_required_param(parse_options(args, "", ["draft"]), github_allowed_subcommands)
-            if not list_args:
-                raise MacheteException(f"`git machete github` expects argument(s): {github_allowed_subcommands}")
-
-            param = args[0]
+            github_subcommand = parsed_cli.subcommand
             machete_client.read_definition_file()
-            if param == "anno-prs":
+            if (github_subcommand in {'anno-prs', 'checkout-prs', 'retarget-pr'} and
+                    'draft' in parsed_cli):
+                raise MacheteException(
+                    "--draft option is NOT valid with 'anno-prs', 'checkout-prs',"
```

Comment:
Also, `pr_no` could be banned for `'anno-prs', 'checkout-prs', 'retarget-pr'`, as for now it's just ignored AFAICS:

```
$ git machete github anno-prs 12345
<OK>
```